internal/postrs/proof.go

```diff
@@ -138,7 +166,14 @@ func (v *Verifier) VerifyProof(
 	k1, k2, k3 uint32,
 	powDifficulty [32]byte,
 	scryptParams ScryptParams,
+	options ...PostOptionFunc,
```

Comment:
Same as for proof generation, would it make sense to change the other parameters too?